config/application.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'boot'
+
+require "rails"
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_mailbox/engine"
+require "action_text/engine"
+require "action_view/railtie"
+require "action_cable/engine"
+
+Bundler.require(*Rails.groups)
+
+module MovieSelectorApiJune2020
+  class Application < Rails::Application
+    config.generators do |generate|
+      generate.helper false
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+  end
+
```

Comment:
These generators has to do with Rspec and should be added when you configure that, not in the scaffold.